imx-core-sdk-kotlin-jvm/src/main/kotlin/com/immutable/sdk/workflows/WithdrawalWorkflow.kt

```diff
@@ -176,6 +166,6 @@ internal data class CompleteWithdrawalWorkflowParams(
     val address: String,
     val starkKey: String,
     val assetType: BigInteger,
-    val amount: BigInteger,
```

Comment:
Can this be removed?

Reply:
It's not used anywhere

Reply:
Will remove instead of commenting it out